tests/e2e/cucumber/features/smoke/copyMove.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Copy
+  As a user
+  I want to copy/move
+  So that I can work safely on a copy or move resource to different location
+
+  Scenario: Copy and move resources in personal space
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+    And "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" creates the following resources
+      | resource                | type    | content                             |
+      | PARENTCopy1             | folder  |                                     |
+      | PARENTCopy2             | folder  |                                     |
+      | PARENTMove              | folder  |                                     |
+      | PARENTCopy3/example.txt | txtFile | example text                        |
+      | KeyboardExample.txt     | txtFile | copy with the help of keyboard      |
+      | dragDrop.txt            | txtFile | copy with the help of drag-drop     |
+      | sidebar.txt             | txtFile | copy with the help of sidebar panel |
+    And "Alice" copies the following resource using sidebar panel
+      | resource    | to          |
+      | sidebar.txt | PARENTCopy2 |
+    When "Alice" copies the following resource
```

Comment:
Maybe it makes sense change as `When "Alice" copies the following resource using dropdownMenu`